docs/storage-abstraction.md

```diff
@@ -1,35 +1,57 @@
+This page explains how to use the Storage API for users who are experts in ScalarDB.
 
-However, directly using the Storage API or mixing the Transaction API and the Storage API could cause unexpected behaviors.
-For example, since the Storage API cannot provide transaction capability, it could cause data inconsistencies/anomalies when failures occur during the execution of the operations.
-Therefore, you should be very careful about using the Storage API, and please use it only if you know exactly what you are doing.
+One of the keys to achieving storage-agnostic or database-agnostic ACID transactions on top of existing storage and database systems is the storage abstraction capabilities that ScalarDB provides. Storage abstraction defines a [data model](design.md#data-model) and the APIs (Storage API) that issue operations on the basis of the data model.
 
-In this document, we explain how to use the Storage API for users who are experts in ScalarDB.
+Although you will likely use the [Transactional API](api-guide.md#transactional-api) in most cases, another option is to use the Storage API.
+
+The benefits of using the Storage API include the following:
+
+- As with the Transactional API, you can write your application code without worrying too much about the underlying storage implementation.
+- If you don't need transactions for some of the data in your application, you can use the Storage API to partially avoid transactions, which results in faster execution.
+
+{% capture notice--warning %}
+**Attention**
+
+Directly using the Storage API or mixing the Transactional API and the Storage API could cause unexpected behavior. For example, since the Storage API cannot provide transaction capability, the API could cause anomalies or data inconsistency if failures occur when executing operations.
+
+Therefore, you should be *very* careful about using the Storage API and use it only if you know exactly what you are doing.
+{% endcapture %}
+
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
 
 ## Storage API Example
 
-This section explains how the Storage API can be used in a simple electronic money example application.
+This section explains how the Storage API can be used in a basic electronic money application.
+
+{% capture notice--warning %}
+**Attention**
+
+The electronic money application is simplified for this example and isn’t suitable for a production environment.
+{% endcapture %}
+
+<div class="notice--warning">{{ notice--warning | markdownify }}</div>
 
 ### ScalarDB configuration
 
-The configuration is the same as when you use the ACID transaction manager.
-Please see [Getting Started](getting-started-with-scalardb.md) for the details of the configuration.
+Before you should configure ScalarDB in the same way mentioned in [Getting Started with ScalarDB](getting-started-with-scalardb.md).
```

Comment:
I feel there seems something wrong in the sentence. (e.g., `Before` is needed?)

Reply:
Ah, the beginning of this sentence is definitely missing something. How does this look?:

```suggestion
Before you begin, you should configure ScalarDB in the same way mentioned in [Getting Started with ScalarDB](getting-started-with-scalardb.md).
```